app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
Toán tử `||=` ý nghĩa như nào và có tác dụng gì ở đây?

Reply:
- Ý nghĩa: kiểm tra giá trị của biến @current_user là true or false(nil)
  + Nếu true => trả về giá trị của biến @current _uesr cho method current_user
  + Nếu false(nil) => gán giá trị của biểu thức User.find_by id: ssession[:user_id] cho biến @current_user. Sau đó trả về kết quả cho method current_user
- ||=  có tác dụng : ghi nhớ kết quả của một hàm để tăng tốc ứng dụng.